src/dialects/abstract/query-interface.d.ts

```diff
@@ -86,9 +86,8 @@ export interface TableNameWithSchema {
   tableName: string;
   schema?: string;
   delimiter?: string;
-  as?: string;
-  name?: string;
```

Comment:
noise added by `addSchema`. I removed it altogether and it has not been a problem. Probably legacy?

Reply:
I think they were used for the alias (`FROM "table" AS "t"`). The only function that was using these properties was `quoteTable`, I made it throw if either property was specified and ran all tests - no issues at all.